draft-ietf-quic-recovery.md

```diff
@@ -1001,14 +1001,21 @@ packets might cause the sender's bytes in flight to exceed the congestion window
 until an acknowledgement is received that establishes loss or delivery of
 packets.
 
+## Persistent Congestion
+
 When an ACK frame is received that establishes loss of all in-flight packets
-sent prior to a threshold number of consecutive PTOs (pto_count is more than
-kPersistentCongestionThreshold, see {{cc-consts-of-interest}}), the network is
-considered to be experiencing persistent congestion, and the sender's congestion
-window MUST be reduced to the minimum congestion window (kMinimumWindow).  This
-response of collapsing the congestion window on persistent congestion is
-functionally similar to a sender's response on a Retransmission Timeout (RTO) in
-TCP {{RFC5681}}.
+sent over a long enough period of time, the network is considered to be
+experiencing persistent congestion.  Commonly, this can be established by
+consecutive PTOs (pto_count is more than  kPersistentCongestionThreshold, see
```

Comment:
```suggestion
consecutive PTOs (pto_count is more than kPersistentCongestionThreshold, see
```

Reply:
Also, I'd remove the () here and then define what the actual period is at the end of this paragraph.